<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>770</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>600</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>194</red>
        <green>194</green>
        <blue>194</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>194</red>
        <green>194</green>
        <blue>194</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>194</red>
        <green>194</green>
        <blue>194</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Распоряжение о финансирование НТП</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="pfin">
           <property name="minimumSize">
            <size>
             <width>470</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>470</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Суммарное плановое финансирование</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="pfin_browser">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="ffin">
         <property name="minimumSize">
          <size>
           <width>470</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>470</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Суммарное фактическое планирование</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="ffin_browser">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="ffin_pfin">
         <property name="minimumSize">
          <size>
           <width>470</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>470</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Доля фактического финансирования от планового (%)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="ffin_pfin_browser">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="kvartal">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>470</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>470</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Квартал</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="kvartal_comboBox">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="input_sum">
         <property name="minimumSize">
          <size>
           <width>470</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>470</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Введите сумма к распределению в текущем квартале </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="input_sum_edit">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="per_sum">
         <property name="minimumSize">
          <size>
           <width>470</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>470</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Введите долю от планового финансирования в текушем квартале (%)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="per_sum_edit">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="ffin_kvartal">
         <property name="minimumSize">
          <size>
           <width>470</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>470</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Фактическое финансирование в текущем квартале </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="ffin_kvartal_browser">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="pfin_kvartal">
         <property name="minimumSize">
          <size>
           <width>470</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>470</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Плановое финансирование в текущем квартале </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="pfin_kvartal_browser">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="proj_label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="check_table_button">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Предпросмотр</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QTableWidget" name="table">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>330</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>370</width>
           <height>200</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="rowCount">
          <number>1</number>
         </property>
         <property name="columnCount">
          <number>3</number>
         </property>
         <row/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_55">
       <item>
        <spacer name="horizontalSpacer_59">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="save">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="save_excel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Сохранить в Excel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_60">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Закрыть</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
